Simplenote.xcodeproj/xcshareddata/xcschemes/Simplenote.xcscheme

```diff
@@ -29,7 +29,8 @@
       shouldUseLaunchSchemeArgsEnv = "YES">
       <Testables>
          <TestableReference
-            skipped = "NO">
+            skipped = "NO"
+            testExecutionOrdering = "random">
```

Comment:
This should help show whether the hanging we're experiencing in `SignupRemoteTests` is due to the test case itself or is perhaps something that happens after n seconds of the tests launch.

If it's due to the test case, it will keep hanging when running it, regardless of its position in the randomly ordered list of tests. Otherwise, we should expect the tests to hang on random test cases.

Reply:
Should we try only with this option first, without skipping a test?

Reply:
Great idea, thanks!

Done in #970.